### DISCUSSION TITLE
How to completely set the camera parameters?

### DISCUSSION THREAD

**Discussion by stla** [pyvista/pyvista#1737] [Q&A] [answered]
Hello,

  I've done the animation shown below. I set the same camera position and the same camera view angle for each frame:

```
    pltr.camera.position = (10.7, 5.7, 5.7)
    pltr.camera.view_angle = 30.0
```

But it seems this is not enough. The solid in the animation (a truncated tesseract) abnormally moves. So it seems that the camera moves. Which camera parameters do we have to set in order to have a completely fixed camera? The position, the direction and the focal point?

![StereoTruncatedTesseract](https://user-images.githubusercontent.com/4466543/138070430-096eab6e-5a07-4211-8099-d1d25da541de.gif)

**Comment by stla** [comment]
I added an invisible circle around the tesseract and this seems good now:

![StereoTruncatedTesseract_b](https://user-images.githubusercontent.com/4466543/138096366-9c1e212c-c7b6-4dda-b550-5d7400cb32bd.gif)

**Comment by MatthewFlamm** [comment, marked as answer]
Look at `set_focus`.  I'm a bit confused by the docstrings between `set_focus` and `set_position`, but I'm assuming the latter is the camera position and the former is to where the camera is pointing.

**Comment by stla** [comment]
I tested with another animation. This seems to work. But it's not easy to find the good values of these parameters. Thank you!

![StereoDuoprism_12-3](https://user-images.githubusercontent.com/4466543/138275023-c3183afa-4b8f-44ce-88bb-de348fc11f2e.gif)
